driver-core/src/main/com/mongodb/client/model/changestream/UpdateDescription.java

```diff
@@ -31,18 +33,37 @@
 public final class UpdateDescription {
     private final List<String> removedFields;
     private final BsonDocument updatedFields;
+    @Nullable
+    private final List<TruncatedArray> truncatedArrays;
 
     /**
      * Creates a new instance
      *
      * @param removedFields the list of fields that have been removed.
      * @param updatedFields the updated fields
+     * @deprecated Not part of the public API since 4.3, may be removed in the future.
+     */
+    @Deprecated
+    public UpdateDescription(@Nullable final List<String> removedFields,
```

Comment:
We cannot remove this constructor because from the user standpoint it was part of the public API. The reason it had to be `public` is that this is required by the BSON codec functionality, but it could have been documented as not being part of the public API.

Reply:
If users can consume these types in their applications, then I think being able to easily create them for their testing scenarios is important. 

For that reason I'm not against the constructors being public and possibly wouldn't even deprecate this method, just have it call the newer constructor.

Reply:
Done.